Server/src/routes/authenticationRouter.ts

```diff
@@ -25,10 +25,11 @@ router.get('/resetPassword', (request: Request, response: Response, next: NextFu
 
 });
 
-router.post('/updateUserInfo', (request: Request, response: Response, next: NextFunction) => {
-    //make an updatePasswordRequest and validatePasswordRequest 
+router.post('/updateUserInfo', JWTAuthenticator.verifyJWT, async (request: Request, response: Response, next: NextFunction) => {
```

Comment:
Could you test this again. To make sure it is functional.

Steps
1 -> Login with valid user 
2 -> Take the Token response and input in Postman
3 -> Find Authorization Tab, select Bearer Token and input the Token Response
4 -> Call your endpoint again

Reply:
managed to get it working after removing the quotation marks from the token response. thank you!